content/2022/simulacija-muve.md

```diff
@@ -4,53 +4,80 @@ summary: Simulacija muve je projekat rađen na letnjem kampu za stare polaznike
 ---
 
 ### Apstrakt
-- 
+
+
 ### Apstrakt na engleskom
-- 
+
+
 ### Uvod
--
-- Sada kada imamo fizičku simulaciju muve, na red je došlo da je naučimo da leti. Naš cilj je bio da muva postigne što veću visinu za određeno vreme.
-Za treniranje muve sam odlučio da koristim genetski algoritam u kombinaciji sa veštačkom neuralnom mrežom.
-- Odlučio sam da iskucam sam svoju neuralnu mrežu, kao i genetski algoritam. Jedina biblioteka koju sam koristio je NumPy da mi omogući rad sa matricama, vektorima i ostale matematičke operacije.
+
+Cilj projekta "Simulacija muve" je da implementiramo fizičku simulaciju i unutar nje pomoću ML-a naučimo muvu da leti.
+Za učenje muve koristimo neuralnu mrežu, napisanu u potpunosti u NumPy-u,  u kombinaciji sa genetskim algoritmom.
```

Comment:
Za sam izveštaj, a posebno za uvodni deo, nije toliko relevantno to što ste kod pisali koristeći NumPy ili neku drugu biblioteku.